testdata/Makefile

```diff
@@ -248,9 +258,9 @@ tlspool-test-client-cert.der: tlspool-test-client-cert.template
 	$(CERTTOOL) --certificate-info --infile $@ --inder --outfile $(@:.der=.pem)
 
 # Key 4: X.509 Server Certificate with user@ domain name
-tlspool-test-server-cert.der: tlspool-test-server-cert.template
+tlspool-test-server-cert.der: tlspool-test-server-cert.template tlspool-test-ca-cert.der
 	echo Using PRIVKEY4, '$(PRIVKEY4)'
-	$(CERTTOOL) --outfile $@ --outder --generate-self-signed --load-privkey='$(PRIVKEY4)' --template=$<
+	$(CERTTOOL) --outfile $@ --outder --generate-certificate --load-ca-certificate=tlspool-test-ca-cert.pem --load-ca-privkey='$(PRIVKEY5)' --load-privkey='$(PRIVKEY4)' --template=$<
 	$(CERTTOOL) --certificate-info --infile $@ --inder --outfile $(@:.der=.pem)
```

Comment:
heb jij dit werkend gekregen zo?

het interfereert met mijn werk om op te tillen naar 0.9.0 maar ik wil dat wel mergen.  maar dan wil ik wel weten dat dit in elk geval op jouw linux systeem werkt.

Reply:
Dit zag je gisteren werken: een server certificaat ondertekend door jouw CA